sklearn/utils/estimator_checks.py

```diff
@@ -283,6 +283,28 @@ def check_estimator(Estimator):
     estimator : estimator object or class
         Estimator to check. Estimator is a class object or instance.
 
+    generate_only : bool, optional (default=False)
+        When `False`, checks are evaluated when `check_estimator` is called.
+        When `True`, `check_estimator` returns a list of checks for the
+        `estimator`.
+
+        .. versionadded:: 0.22
+
+    Notes
+    -----
+    Setting `generate_only=False` allows for checks to be disaseembled when
+    testing an estimator. To use this with pytest this can be done with:
+
+    .. code-block:: python
+
+        from itertools import chain
```

Comment:
```python
import pytest
from sklearn.utils.estimator_check import check_estimator
```

Reply:
@thomasjpfan This wasn't addressed in your last commit..

Reply:
Done.